-Failed tests
Test 1: Teste 1
Test 2: Teste 2
Test 3: Teste 3
-Test 1: Teste 1 (-6.667)
Incorrect program result
 --- Input ---
> 3
>8
>0
>7
>4
>5
>6
>3
>10
>2

 --- Program output ---
>digite a dimensão da matriz:digite um valor:digite um valor:digite um valor:digite um valor:digite um valor:digite um valor:digite um valor:digite um valor:digite um valor:Traceback (most recent call last):
>  File "matriz2.py", line 49, in <module>
>    if quadradomagico(a):
>  File "matriz2.py", line 26, in quadradomagico
>    sd1=somasiagonal1(a)
>NameError: global name 'somasiagonal1' is not defined

 --- Expected output (text)---
> 'S'

-Test 2: Teste 2 (-6.667)
Incorrect program result
 --- Input ---
> 2
>1
>1
>1
>1

 --- Program output ---
>digite a dimensão da matriz:digite um valor:digite um valor:digite um valor:digite um valor:Traceback (most recent call last):
>  File "matriz2.py", line 49, in <module>
>    if quadradomagico(a):
>  File "matriz2.py", line 26, in quadradomagico
>    sd1=somasiagonal1(a)
>NameError: global name 'somasiagonal1' is not defined

 --- Expected output (text)---
> 'S'

-Test 3: Teste 3 (-6.667)
Incorrect program result
 --- Input ---
> 2
>1
>2
>5
>8

 --- Program output ---
>digite a dimensão da matriz:digite um valor:digite um valor:digite um valor:digite um valor:Traceback (most recent call last):
>  File "matriz2.py", line 49, in <module>
>    if quadradomagico(a):
>  File "matriz2.py", line 26, in quadradomagico
>    sd1=somasiagonal1(a)
>NameError: global name 'somasiagonal1' is not defined

 --- Expected output (text)---
> 'N'

-Summary of tests
>+------------------------------+
>|  3 tests run/ 0 tests passed |
>+------------------------------+

